samples/src/commonMain/kotlin/ru/nsk/samples/PlantUmlExportSample.kt

```diff
@@ -10,15 +10,15 @@ private object PlantUmlExportSample {
 }
 
 fun main() = runBlocking {
-    val machine = createStateMachine(this, "Nested states") {
-        val state1 = initialState("State1")
-        val state3 = finalState("State3")
+    val machine = createStateMachine(this, metaInfo = umlLabel("Nested states")) {
```

Comment:
Could you please make separate (copy pasted) sample with metaInfo? As it is not mandatory argument for export feature.